### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values/conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#198]
```
Proofread for typos, grammar, punctuation.  
```

Original issue reported on code.google.com by `medic....@gmail.com` on 3 Jun 2011 at 2:52

Attachments:
- [conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-198/comment-0/conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
The propsed corrections look great. Thanks a lot!

Posting patches is absolutely preferred, like you did. To make the life easier 
for the people reviewing the patches, please run diff with "diff -Naur 
file1.xml file2.xml", which produces a smaller more readable patch file. If you 
run some graphical IDE that produces the diffs, I'm sure there's a switch in 
there somewhere to produce "unified" diffs.

Thank you again! Will commit this patch to v0.6.9.
```

Original comment by `oskar.wi...@gmail.com` on 7 Jun 2011 at 6:41
- Changed state: **Started**
- Added labels: **Type-Translation**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Committed in r101. Thank you again!
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2011 at 9:12
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:53
- Changed state: **Done**
